tests/integration/test_writes/test_writes.py

```diff
@@ -1448,3 +1450,100 @@ def test_rewrite_manifest_after_partition_evolution(session_catalog: Catalog) ->
             EqualTo("category", "A"),
         ),
     )
+
+
```

Comment:
As part of the integration tests, can you also test interacting with Spark? I.e. write branch from pyiceberg, read from spark and vice versa

Reply:
@kevinjqliu Have added integrations tests with spark
Resolving this conversation